clients/client-python/setup.py

```diff
@@ -33,7 +33,7 @@
     version="0.6.0.dev1",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/datastrato/gravitino",
+    url="https://github.com/apache/gravitino",
     author="datastrato",
     author_email="support@datastrato.com",
```

Comment:
@justinmclean can we update these two lines?

Reply:
There's another issue to do that

Reply:
OK.